docs/developer/bcd.rst

```diff
@@ -0,0 +1,97 @@
+=================================================
+BCD - the Bodhi Container Development environment
+=================================================
+
+BCD is an Ansible-orchestrated, Podman-powered container-based development environment for Bodhi.
+It lets you quickly and easily test your changes in a live Bodhi environment, without any external
+dependencies, root privilege requirements, or virtual machines. It's also convenient for running
+the unit tests.
+
+To get started on a Fedora or Enterprise Linux-ish system, do::
+
+    $ sudo dnf install ansible podman
```

Comment:
We generally suggest using ansible-core + the needed collections instead of the large `ansible` package.

```suggestion
    $ sudo dnf install ansible-core ansible-collection-containers-podman podman
```